File: 0976-largest-perimeter-triangle/0976-largest-perimeter-triangle.cpp
 class Solution {
 public:
  int largestPerimeter(vector<int>& A) {
    sort(A.begin(), A.end(), greater<int>());
    for (int i = 0; i < (int)A.size() - 2; ++i)
      if (A[i] < A[i + 1] + A[i + 2])
        return A[i] + A[i + 1] + A[i + 2];
    return 0;
  }
 };

File: 0976-largest-perimeter-triangle/0976-largest-perimeter-triangle.py
class Solution:
    def largestPerimeter(self, nums: List[int]) -> int:
        nums.sort()
        prem =  0
        
        for i in range(2, len(nums)):
            if nums[i-2] + nums[i-1] > nums[i]:
                prem = max(prem ,nums[i-2] + nums[i-1] +nums[i] )
        return prem
        